### PR TITLE
CakeShells should not print html in case of a error #6720

### PR DESCRIPTION
**Pull request by begnini**
I'm open again this PR with @markstory suggestion.

### CODE REVIEW

**Comment by markstory** [comment]
Looks better to me :+1:

File: lib/Cake/Console/ShellDispatcher.php [hunk 1]
```diff
@@ -177,6 +177,9 @@ public function setErrorHandlers() {
 		}
 		set_exception_handler($exception['consoleHandler']);
 		set_error_handler($error['consoleHandler'], Configure::read('Error.level'));
+
+		App::uses('Debugger', 'Utility');
```
Comment (lorenzo) [inline lib/Cake/Console/ShellDispatcher.php]: Can you add this line at the top of the file?

Comment (begnini) [inline lib/Cake/Console/ShellDispatcher.php]: Done!

Comment (begnini) [inline lib/Cake/Console/ShellDispatcher.php]: This don't work. Class App isn't declared until ShellDispatcher::_bootstrap be called :(. I will revert my commit.

**Comment by markstory** [comment]
Would you be able to squash down the noisy commits?

**Comment by markstory** [comment]
Merged into 2.6 in bf82c95d6462252a14922a2885ec4c1a5f6d7109

**Pull request closed without merging**